include/data.h

```diff
@@ -40,6 +40,10 @@ class Metadata {
     /// Find the data set containing the sample.
     ///
     /// The data set may contain other samples.
+    ///
+    /// The resulting data structure may be shared, so the strings must not be
+    /// mutated. They aren't declared const because...C++
+    /// http://stackoverflow.com/a/21365478
```

Comment:
I think I'd recently removed this comment because it was a copy-paste error. It should not be here I think.